core/src/test/java/nl/nn/adapterframework/configuration/digester/ConfigurationDigesterTest.java

```diff
@@ -131,6 +133,39 @@ public void stub4testtoolTest() throws Exception {
 		String expectedConfiguration = TestFileUtils.getTestFile(baseDirectory + "/expected.xml");
 		MatchUtils.assertXmlEquals(expectedConfiguration, actual);
 	}
+	
+	@Test
+	public void stub4testtoolEsbJmsListenerTest() throws Exception {
+		String baseDirectory = "/ConfigurationUtils/stub4testtool/EsbJmsListener";
+
+		StringWriter target = new StringWriter();
+
+		XmlWriter xmlWriter = new XmlWriter(target) {
+			@Override
+			public void comment(char[] ch, int start, int length) throws SAXException {
+				if(!new String(ch).startsWith("<receiver name='receiver' transactionAttribute='Required' transactionTimeout=")) {
+					fail("Digester should have commented out the receiver that has EsbJmsListener");
+				}
+				super.comment(ch, start, length);
+			}
```

Comment:
You should also override startElement() and check that there is not a listener element with className EsbJmsListener